### ISSUE TITLE
Embedding a large amount of popupImages in a map

### ISSUE THREAD

**Issue by adrfantini** [r-spatial/mapview#168]
I have a map which contains several thousands points, each of which should have a different popup image which is a local `.png`. Currently the generation of the map is extremely slow, to the point I kill it after a couple of hours.

Is mapview making a copy of all the images and loading all of them, or is it somehow possible to make it so they are loaded only upon request?

**Comment by tim-salabim** [comment]
We've changed the way local image files are embedded a couple of months ago. Previously, the (e.g.) pngs were copied to a temp folder and then <src=...>'d, now they are embedded via [base64enc::base64encode](https://github.com/r-spatial/mapview/blob/develop/R/popup.R#L141-L147). The previous solution should be a lot faster and performant. We made the change because when savong the map via `saveWidget`/`mapshot` the images were lost as they were not saved in the correct place. 

Maybe we should provide the user with a choice to embed them, but have the sourced pngs as a default? This is a bit of a conundrum as we then again loose the ability to save the images in a straght forward manner... To get them to the correct place the user needs to copy the folder containing the pngs to one level above the index.html file. It is ia mess...

Any thoughts?
